docs/sdk/core/app2.md

```diff
@@ -171,9 +175,6 @@ func (tx app2Tx) GetMsgs() []sdk.Msg {
 }
 ```
 
-We don't need a custom TxDecoder function anymore, since we're just using the
```

Comment:
Why remove this ?

Reply:
The default TxDecoder decodes into auth.StdTx. Since the tx in app2 is of type app2Tx this will fail